albatross/models/ransac_gp.h

```diff
@@ -25,6 +25,17 @@ template <typename FeatureType> struct FitAndIndices {
   Indexer fit_indices;
 };
 
+// Calculate a numerically stable log determinant of a symmetric matrix using Cholesky
+inline double log_determinant_of_symmetric(const Eigen::MatrixXd &M) {
+  double ld = 0;
```

Comment:
nit: I think it's fairly obvious, but perhaps a more descriptive variable name? `log_det`, `log_determinant`? `ld` combined with `ldlt` might get slightly confusing.